mautic/mautic
# TPROD-278 - Do not add format option when html5 is set for DateType

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | https://mautic.atlassian.net/browse/TPROD-278



#### Description:


It is no longer valid to set `format` option for date form types when `html5` is set to `true` in Symfony 5. This PR fixes this.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to any contact detail page.
3. Click the arrow button in the top left corner and select preferences.
4. Make sure there are no issues with the pause from/to date time boxes/pickers.
5. Make sure that pause from/to date time boxes/pickers are also working for the public preference page.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11038"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
